STT/STT_RealTime.py

```
@@ -0,0 +1,32 @@
+
+    
+    def callback(self, indata, frames, time, status):
+        """Этот метод вызывается для каждого блока аудио"""
+        if status:
+            print(status, file=sys.stderr)
+        self.q.put(bytes(indata))
+
+    def get_speech(self):
+        with sd.RawInputStream(samplerate = self.samplerate, blocksize=8000, dtype="int16", channels=1, callback=self.callback):
+            rec = KaldiRecognizer(self.model, self.samplerate)
+            listen = True
```

Ruff (0.8.0)

Local variable `listen` is assigned to but never used

Remove assignment to unused variable `listen`

(F841)